transaction/send_klay_with_keystore_file/src/main/java/Boilerplate.java

```diff
@@ -0,0 +1,114 @@
+    private static String secretAccessKey = ""; // e.g. "aP/reVYHXqjw3EtQrMuJP4A3/hOb69TjnBT3ePKG";
+    private static String chainId = ""; // e.g. "1001" or "8217";
+    private static String recipientAddress = ""; // e.g. "0xeb709d59954f4cdc6b6f3bfcd8d531887b7bd199"
+
+    public static void main(String[] args) {
+        try {
+            loadEnv();
+            run();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static String objectToString(Object value) throws JsonProcessingException {
+        ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
+        return ow.writeValueAsString(value);
+    }
+
+    public static void loadEnv() {
+        Dotenv env;
+        String workingDirectory = System.getProperty("user.dir");
+        Path workingDirectoryPath = Paths.get(workingDirectory);
+        String projectRootDirectory = "caver-java-examples";
+        String currentDirectoryName = workingDirectoryPath.getName(workingDirectoryPath.getNameCount() - 1).toString();
+        String envDirectory = currentDirectoryName.equals(projectRootDirectory) ?
+                workingDirectoryPath.toString() :
+                workingDirectoryPath.getParent().getParent().toString();
+
+        // Read `/path/to/caver-java-examples/.env` file.
+        env = Dotenv.configure().directory(envDirectory).load();
+
+        nodeApiUrl = nodeApiUrl.equals("") ? env.get("NODE_API_URL") : nodeApiUrl;
+        accessKeyId = accessKeyId.equals("") ? env.get("ACCESS_KEY_ID") : accessKeyId;
+        secretAccessKey = secretAccessKey.equals("") ? env.get("SECRET_ACCESS_KEY") : secretAccessKey;
+        chainId = chainId.equals("") ? env.get("CHAIN_ID") : chainId;
+        recipientAddress = recipientAddress.equals("") ? env.get("RECIPIENT_ADDRESS") : recipientAddress;
+    }
+
+    public static void run() throws Exception {
+        HttpService httpService = new HttpService(nodeApiUrl);
+        httpService.addHeader("Authorization", Credentials.basic(accessKeyId, secretAccessKey));
+        httpService.addHeader("x-chain-id", chainId);
+        Caver caver = new Caver(httpService);
+
+        // 1. Create your own keystore file at "https://baobab.wallet.klaytn.com/create".
+        // 2. Rename that keystore file name with `keystore.json` or change the filename in line 81 and 84.
+        // 3. Place that keystore file at `caver-java-examples/kct/deploy_kip7_token_contract_with_keystore_file/resources`.
+        // 4. Get 5 KLAY at "https://baobab.wallet.klaytn.com/faucet".
+        File file = new File("resources/keystore.json");
+        if(file.exists() == false) {
+            // Handles when you run this Boilerplate as sub-module using IDE.
+            file = new File("transaction/send_klay_with_keystore_file/resources/keystore.json");
+            if(file.exists() == false) {
+                throw new Exception("Cannot find keystore.json file.");
```

Comment:
```suggestion
                throw new Exception("Cannot find keystore.json file.");
```

Please throw meaningful exception.

Reply:
@sirano11 Thanks for your feedback.
I'll update this PR soon :)

Reply:
I updated it [here](https://github.com/klaytn/caver-java-examples/pull/7/commits/373b2359b3940d32c721fd14d10671f964f71683) :)